Tests/pens/ttGlyphPen_test.py

```diff
@@ -290,6 +290,310 @@ def test_scaled_component_bounds(self):
         self.assertGlyphBoundsEqual(uni0302_uni0300, (-278, 745, 148, 1025))
 
 
+class TTGlyphPointPenTest:
+    def runEndToEnd(self, filename):
+        font = ttLib.TTFont()
+        ttx_path = os.path.join(
+            os.path.abspath(os.path.dirname(os.path.realpath(__file__))),
+            "..",
+            "ttLib",
+            "data",
+            filename,
+        )
+        font.importXML(ttx_path)
+
+        glyphSet = font.getGlyphSet()
+        glyfTable = font["glyf"]
+        pen = TTGlyphPointPen(font.getGlyphSet())
+
+        for name in font.getGlyphOrder():
+            oldGlyph = glyphSet[name]
+            oldGlyph.drawPoints(pen)
+            oldGlyph = oldGlyph._glyph
+            newGlyph = pen.glyph()
+
+            if hasattr(oldGlyph, "program"):
+                newGlyph.program = oldGlyph.program
+
+            assert oldGlyph.compile(glyfTable) == newGlyph.compile(glyfTable)
+
+    def test_e2e_linesAndSimpleComponents(self):
+        self.runEndToEnd("TestTTF-Regular.ttx")
+
+    def test_e2e_curvesAndComponentTransforms(self):
```

Comment:
these methods are intentical for TTGlyphPenTest and TTGlyphPointPenTest classes, use the same trick that I suggested for runEndToEnd above, and define them in the base class then subclass from them, the test will be run with both using different pen classes 

Only make sure you don't name the base class with `*Test` suffix otherwise pytest thinks it is a concrete test class (e.g. like `TTGlyphPenTest`). Call it "TTGlyphPenTestBase" maybe